src/components/animated/_nudge.less

```diff
@@ -1,6 +1,12 @@
+.nudgeAnimation() {
+  animation: nudge 0.75s;
+}
 .nudge-btn {
   &:hover {
-    animation: nudge 0.75s;
+    .nudgeAnimation();
+  }
+  &.animated {
```

Comment:
Combine the `&.animated` with `&:hover` and only add in `&.animated` the following:

```
    animation-iteration-count: infinite;
```
THis is to make the animation continuous

Reply:
I made the changes.